Matlab/Matlab.sublime-syntax

```diff
@@ -62,7 +62,7 @@ contexts:
       scope: comment.double.percentage.matlab
       captures:
         1: punctuation.definition.comment.matlab
-    - match: '%\{'
+    - match: '%\{\s*\n'
```

Comment:
Though not changed by you, the `match` on line 72 should have `()` around the `%\}` also.